manifests/pipecd/values.yaml

```diff
@@ -274,6 +274,8 @@ tempo:
   # modified by @pipecd, to expand template remoteWriteUrl
   # @default -- Dynamically generated tempo configmap
   config: |
+    memberlist:
+      cluster_label: "{{ .Release.Name }}.{{ .Release.Namespace }}"
```

Comment:
This change follows the upstream change.
https://github.com/grafana/helm-charts/commit/e66f68af81f9eb069c13843dfc0bffdea9027aee